expcore/Mini_games.lua

```diff
@@ -77,6 +74,14 @@ end
 function Mini_games._prototype:add_stop_function(stop_function)
     self.stop_function = stop_function   
 end
+function Mini_games._prototype:add_gui_element(gui_element)
+    self.gui = gui_element   
+end
+
+function Mini_games._prototype:add_gui_callback(callback)
+    self.gui_callback = callback   
+end
```

Comment:
I know that you have used `add` for your other functions; but, would it not make more sense to call them `set` after all your are setting a value rather than adding to a list. Maybe you want to change this for all functions that are like this?

Reply:
yeah i gues that does make more sence